context-free.md

```diff
@@ -2,21 +2,42 @@
 
 ##  Who is this programming language for?
 
+This programming language is for people who want to create art on the computer 
+through rules. Instead of using a mouse to point and click like you would in 
+paint, you rather create mathematical rules that creates art.
 
 ## What is easy to do in this language? Why is it easy?
 
+It is easy to do recursive shapes. It is easy because the language allows you 
+to define rules with itself.
 
```

Comment:
I agree with you on that Context Free makes recursion easy by allowing you to call the rule while defining it.
This seems to contribute to it being "domain-specific", since this mainly helps with creating shapes recursively, which is something that might be hard to do in some general-purpose languages.